crypto/rand/rand_lcl.h

```diff
@@ -106,8 +114,24 @@ struct rand_drbg_st {
     size_t min_entropylen, max_entropylen;
     size_t min_noncelen, max_noncelen;
     size_t max_perslen, max_adinlen;
-    unsigned int reseed_counter;
+    /* counts the number of generate requests since the last reseed */
+    unsigned int generate_counter;
+    /* maximum number of generate requests until a reseed is required */
     unsigned int reseed_interval;
+    /* counts the number of reseeds since instantiation */
+    unsigned int reseed_counter;
```

Comment:
Can you add that that is for the parent and that for the child it's the parent's reseed counter?

Reply:
Yes, I will add a more precise explanation.